<ui version="4.0" >
 <class>kateCtags</class>
 <widget class="QWidget" name="kateCtags" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>132</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>0</number>
   </property>
   <item row="0" column="0" >
    <widget class="KTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="hits" >
      <attribute name="title" >
       <string>Tags</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2" >
       <item row="0" column="0" >
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Lookup:</string>
         </property>
         <property name="buddy" >
          <cstring>input_edit</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="KLineEdit" name="input_edit" >
         <property name="showClearButton" stdset="0" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2" >
        <widget class="QTreeWidget" name="treeWidget" >
         <property name="rootIsDecorated" >
          <bool>false</bool>
         </property>
         <column>
          <property name="text" >
           <string>Tag</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Type</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>File</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="settings" >
      <attribute name="title" >
       <string>Settings</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3" >
       <item row="0" column="0" >
        <widget class="QLabel" name="cmdLabel" >
         <property name="layoutDirection" >
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text" >
          <string>CTags Command</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="4" >
        <widget class="KLineEdit" name="cmdEdit" />
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="fileLabel" >
         <property name="layoutDirection" >
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text" >
          <string>CTags Database File</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="KLineEdit" name="tagsFile" >
         <property name="showClearButton" stdset="0" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="2" >
        <widget class="KPushButton" name="loadFile" >
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="3" >
        <widget class="KPushButton" name="updateDB" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="4" >
        <widget class="KPushButton" name="newDB" >
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <spacer name="verticalSpacer" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>1</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KTabWidget</class>
   <extends>QTabWidget</extends>
   <header>ktabwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
